_posts/2020-04-28-history-of-bel.md

```diff
@@ -169,6 +211,8 @@ hosts its first [BEL-specific text mining challenge](https://biocreative.bioinfo
 It was organized by OntoGene (Fabio Rinaldi), the sbv IMPROVER/PMI (Sam Ansari, Julia Hoeng), and Fraunhofer
 (Juliane Fluck, Martin Hofmann Apitius) following the footsteps of the sbvIMPROVER network verification challenge.
 
+**March 2015** David de Graaf steps out of CEO role at Genstruct, joins Flagship Venturelabs. 
```

Comment:
Do you know who becomes the last CEO? I'm also curious if you have anything you'd like to add about Selventa's dissolution.